blog/2024-09-24-robotflow-reductstore/index.mdx

```diff
@@ -0,0 +1,397 @@
+
+- every nth record (e.g., every 10th record)
+- at specific intervals, such as every s-th second (e.g., every 5 seconds).
+
+To set up replication, we can use the ReductStore web console to define the replication source and destination as well as rules such as the replication interval and filtering criteria.
+
+![ReductStore Data Replication](./img/reductstore_data_replication.png)
+
+<small style={{ display: "block", marginTop: "-20px" }}>
+  ReductStore Data Replication. Image source: [ReductStore](/)
+</small>
+
+For more information on selective data replication to the cloud, refer to the [**ReductStore replication guide**](https://www.reduct.store/docs/next/guides/data-replication).
+
+### Querying Data from ReductStore
+
+Once your data is stored in ReductStore, retrieving it for analysis or further processing is easy. You can query and retrieve data based on the timestamp used during storage or for a specific time range.
+
+Here is an example of how you would query data from ReductStore using the Python client for a specific timestamp:
+
+```python
+async with Client("http://127.0.0.1:8383", api_token="my-token") as client:
+    bucket: Bucket = await client.get_bucket("my-bucket-1")
+
+    async with bucket.read("roboflow", ts) as record:
+        content = await record.read_all()
+        image_reconstructed = bytes_to_image(content)
+        label_reconstructed = unflatten_predictions(record.labels)
+
+        assert np.array_equal(image, image_reconstructed)
+```
+
+In this example, the `bucket.read` method retrieves the data using the same entry name (`"roboflow"`) and timestamp that was used during the storage process. The stored image can be reconstructed from the bytes, and the corresponding AI-generated labels can be recovered from the metadata.
+
+As mentioned above, ReductStore supports querying data over a time range, so you can retrieve data between specific timestamps as needed. For more details on query options, please refer to the [**ReductStore documentation**](https://www.reduct.store/docs/guides/data-querying).
+
+## Conclusion
+
+The combination of Roboflow and ReductStore provides a comprehensive solution for efficiently deploying and managing computer vision models. Roboflow simplifies the model development and deployment process and makes it accessible to developers and data scientists, while ReductStore ensures that the massive amounts of data generated by these models are effectively managed, especially in real-time environments such as video streams.
+
+Whether deployed in the cloud or at the edge, these tools reduce operational complexity, lower costs, and provide the flexibility to scale computer vision applications. As computer vision continues to evolve, leveraging these platforms will be essential to building efficient and scalable solutions.
+
+## Frequently Asked Questions (FAQs)
+
```

Comment:
does it makes sens to add a question about  ReductStoer cloud solution and point to the azure VM and guide?